### DISCUSSION TITLE
The model management is at CLI but not API Server

### DISCUSSION THREAD

**Discussion by vansangpfiev** [janhq/nitro#1386] [Team Discussions]
Currently, CLI and API Server can directly access model data, which means we support APIs Server for model management, but CLI only uses some of them. Should CLI only use underlying APIs?

IMO, we are building only 1 execution for both CLI and Server. Therefore, it makes sense when we use CLI without calling APIs. If CLI only uses underlying APIs, should we build 2 executable binaries, each for CLI and Server?
More context: 
https://discordapp.com/channels/1107178041848909847/1204617960615055401/1290548201186660362
cc: @dan-homebrew @louis-jan @namchuai @nguyenhoangthuan99

**Comment by dan-menlo** [comment]
@vansangpfiev Will this be similar to Docker?

- docker-cli
- docker-engine

Will there be any impact on our packaging and installer, if we move to two binaries?

**Reply by vansangpfiev** [reply]
> @vansangpfiev Will this be similar to Docker?
> 
> * docker-cli
> * docker-engine
> 
> Will there be any impact on our packaging and installer, if we move to two binaries?

I believe it will impact the packaging, installer, and updater logic also  cc: @hiento09

**Reply by dan-menlo** [reply]
I see. Let's keep this for v1.0.1 then. Let's ship the existing version for v1.0.0.
